coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -113,6 +127,14 @@ def _parse_documentation_with_symbols(self,
             section is a named tuple of either ``Description``, ``Parameter``,
             ``ExceptionValue`` or ``ReturnValue``.
         """
+        def _reference(line, ref_identifiers):
+            occurences = []
+            for ref_type in ref_identifiers.keys():
+                ref = line.find(ref_identifiers[ref_type][0])
```

Comment:
what if the same ref type is used twice in the same line?

Reply:
Thank You for the tricky catch. Making changes.